cmd/server/command/flags.go

```diff
@@ -0,0 +1,82 @@
+package command
```

Comment:
I think the file name should be changed or will it handle other stuff than Grafana?

Reply:
I'm planning on moving the other custom flags over here as well :)